.env.compose

```
@@ -557,7 +557,36 @@ MCP_WS_TLS=false
+MCP_AUTH_RESOURCE_URI=https://mcp.gauzy.co
+
+# JWT validation for production (using RS256 with public key)
+MCP_AUTH_JWT_ALGORITHMS=RS256,ES256
+MCP_AUTH_JWT_AUDIENCE=https://mcp.gauzy.co
+MCP_AUTH_JWT_ISSUER=https://auth.gauzy.co
+
+# Option 1: Use JWKS URI for dynamic key discovery (recommended)
+MCP_AUTH_JWT_JWKS_URI=https://auth.gauzy.co/.well-known/jwks.json
+
+# Production authorization server configuration
+MCP_AUTH_SERVERS=[{"issuer":"https://authdemo.gauzy.co","authorizationEndpoint":"https://authdemo.gauzy.co/oauth2/authorize","tokenEndpoint":"https://authdemo.gauzy.co/oauth2/token","registrationEndpoint":"https://authdemo.gauzy.co/oauth2/register","introspectionEndpoint":"https://authdemo.gauzy.co/oauth2/introspect","grantTypesSupported":["authorization_code","refresh_token","client_credentials"],"responseTypesSupported":["code"],"scopesSupported":["mcp.read","mcp.write","mcp.admin","openid","profile","email"],"codeChallengeMethodsSupported":["S256"]}]
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)